pkg/cmd/roachtest/tests/mixed_version_tenant_span_stats.go

```diff
@@ -193,16 +148,20 @@ func fetchSpanStatsFromNode(
 	return &res, nil
```

Comment:
While we're here, we can also update this `Printf` call to avoid the `%!(EXTRA string` formatting warning:

```go
l.Printf("fetchSpanStatsFromNode - curl result:\n%s", res.Stdout)
```

Reply:
Where are you seeing this formatting warning?

Reply:
https://github.com/cockroachdb/cockroach/blob/d30bce58d9a75377df3f2cb00b3a36c0e34c6e2f/pkg/cmd/roachtest/tests/mixed_version_tenant_span_stats.go#L192

Reply:
Ah, facepalm. I didn't notice the %s placeholder formatting in your previous comment.